spec/models/miq_report/seeding_spec.rb

```diff
@@ -94,6 +94,17 @@
 
         expect { report.reload }.to  raise_error(ActiveRecord::RecordNotFound)
         expect { compare.reload }.to raise_error(ActiveRecord::RecordNotFound)
+
+        # Duplicate custom reports by name will skip seeding
```

Comment:
I would change this to something like:

```ruby
# Duplicate custom reports by name will be skipped in seeding
```

Your version makes it seem like seeding will be skipped all together.